### ISSUE TITLE
Javascript poll column updating?

### ISSUE THREAD

**Issue by naught101** [kellerben/dudle#64]
I'm a regular user of https://dudle.inf.tu-dresden.de/ - it's much more useful than most other similar services. The only problem is that it's *really* slow when creating date/time polls with lots of options. You have to click and wait for a page load for every single date/time addition (which can take 5 seconds, if the server's having a bad day), and if you decide you screwed up and put the wrong date in, you need to click and wait to remove that too. A poll with 30-40 options can take 10 minutes to complete properly.

It would be really nice if there was some faster way to add/edit the poll data. Perhaps using Javascript to update the form, and then having a separate "save" button, to actually send the form back to the server?

**Comment by kellerben** [comment]
I agree, could be solved using a dudle-extension as well (patches welcome!)

**Comment by valioDOTch** [comment]
I'm a frontend dev and I would really like to see this fixed.
Which docker container should I prefer?
If I'm mainly interested in writing an extension that injects JS into the html body, how should I proceed? Is there something like a skeletton extension that I could clone?

**Comment by kellerben** [comment]
try this extension: https://github.com/kellerben/dudle-extensions-participate
It is already the common extension for symcrypt, asymcrypt and gpgauth. If you look at this example:
https://dudle.inf.tu-dresden.de/coffeebreak-symcrypt/#passwd=Ai8yeebi
you will discover, that requests are done via javascript already. In common.js is a webservice call to the store service. It is currently only used for voting but could be used for configuration of the poll as well.

**Comment by valioDOTch** [comment]
Thanks for the additional info @kellerben.
I will try to take a look at it ASAP.
That might be by the end of the year however, as things often heat up for me at the beginning of December.
